submit/submit

```diff
@@ -0,0 +1,546 @@
+for index, filename in enumerate(args.filename):
+    # Ignore doubly specified files
+    if filename in filenames:
+        logging.debug(F"ignoring doubly specified file `{filename}'")
+        continue
+
+    # Stat file and do some validation checks
+    try:
+        st = os.stat(filename)
+    except FileNotFoundError:
+        usage(F"Can not find file `{filename}'")
+    
+    logging.debug(F"submission file {index + 1}: `{filename}'")
+
+    # Do some checks on submission file and warn user
+    if not stat.S_ISREG(st.st_mode):
+        warn_user(F"`{filename}' is not a regular file")
+    if not st.st_mode & stat.S_IRUSR:
+        warn_user(F"`{filename}' is not readable")
+    if st.st_size == 0:
+        warn_user(F"`{filename}' is empty")
+
+    file_age = (time.time() - st.st_mtime) / 60
+    if file_age > warn_mtime_minutes:
+        warn_user(F"`{filename}' has not been modified for {int(file_age)} minutes")
+
+    if magic:
+        m = magic.from_file(filename, mime=True)
+        if m[:5] != 'text/':
+            warn_user(F"`{filename}' is detected as binary/data")
+
+    filenames.append(filename)
+
+# Try to parse problem and language from first filename
+
+filebase = os.path.basename(filenames[0])
+
+if '.' in filebase:
+    dot = filebase.rfind('.')
+    ext = filebase[dot+1:]
+    filebase = filebase[:dot]
+
+    if not problem_id:
+        problem_id = filebase
+    if not language_id:
+        language_id = ext
+
+# Check for languages matching file extension
+language_id = language_id.lower()
+for language in languages:
+    for extension in language['extensions']:
+        if extension.lower() == language_id:
+            my_language = language
+            break
+    if my_language:
+        break
+
+if not my_language:
+    usage('no known language specified or detected')
+
+# Check for problem matching ID or label
+problem_id = problem_id.lower()
+for problem in problems:
+    if problem['id'].lower() == problem_id or problem['label'].lower() == problem_id:
+        my_problem = problem
+        break
+
+if not my_problem:
+     usage('no known problem specified or detected')
+
+# Guess entry point if not already specified.
+if not entry_point and my_language['entry_point_required']:
+    if my_language['name'] == 'Java':
+        entry_point = filebase
+    elif my_language['name'] == 'Kotlin':
+        entry_point = kotlin_base_entry_point(filebase) + "Kt"
+    elif my_language['name'] == 'Python 3':
+        entry_point = filebase + "." + ext
+
+if not entry_point and my_language['entry_point_required']:
+    error('Entry point required but not specified nor detected.')
+
+logging.debug(F"contest is `{my_contest['shortname']}'")
+logging.debug(F"problem is `{my_problem['label']}'")
+logging.debug(F"language is `{my_language['name']}'")
+logging.debug(F"entry_point is `{entry_point or '<None>'}'")
+logging.debug(F"url is `{baseurl}'")
+
+if not args.assume_yes:
+    print('Submission information:')
```

Comment:
```suggestion
    global num_warnings
    print('Submission information:')
```

Reply:
This is not in a method but already in a global scope right? Does this even work?

Reply:
You're right, but than the LGTM message makes little sense, I assumed function due to indentation.

Reply:
Yeah I don't get LGTM either

Reply:
DNLGTM